blobfuse/blobfuse.cpp

```diff
@@ -490,19 +496,24 @@ void print_version()
     fprintf(stdout, "blobfuse 1.2.3\n");
 }
 
-int set_log_mask(const char * min_log_level_char)
+int set_log_mask(const char * min_log_level_char, bool blobfuseInit)
 {
     if (!min_log_level_char)
     {
+        syslog(LOG_CRIT, "Setting logging level to : LOG_WARNING");
         setlogmask(LOG_UPTO(LOG_WARNING));
         return 0;
     }
     std::string min_log_level(min_log_level_char);
     if (min_log_level.empty())
     {
+        syslog(LOG_CRIT, "Setting logging level to : LOG_WARNING");
         setlogmask(LOG_UPTO(LOG_WARNING));
         return 0;
     }
+    
+    syslog(LOG_CRIT, "Setting logging level to : %s", min_log_level.c_str());
```

Comment:
This should not be LOG_CRIT , just LOG_INFO because we are just setting the parameter specified by the user.

Reply:
Kept it intentionally to debug any issue where we received the signal but did not change the logging level. If it goes to log_info then we may not get what logging level was set in code initially or during signal handling.